index.js

```diff
@@ -73,25 +73,48 @@ function toDateString(when) {
 
 // Linkify `message` string and add it to `parent` node.
 function linkifyAndAdd(parent, message) {
-    while (true) {
-        const matchBugNumber = message.match(/bug (\d+)/i);
-        if (matchBugNumber === null) {
-            const text = DOM.createText(message);
-            parent.appendChild(text);
-            break;
+    const searches = [
+        {
+            regexp: /bug (\d+)/i,
+            createLink: match => urls.bug(match[1])
+        },
+        {
+            // Shamelessly taken from Stackoverflow:
+            // https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url
+            regexp: /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/i,
+            createLink: match => match[0]
         }
+    ];
 
-        const [matched, bugNumber] = matchBugNumber;
-        const beforeText = message.substr(0, message.indexOf(matched));
-        const afterText = message.substr(message.indexOf(matched) + matched.length, message.length);
+    let shouldContinue = true;
+    while (shouldContinue) {
+        shouldContinue = false;
 
-        parent.appendChild(DOM.createText(beforeText));
+        for (let search of searches) {
+            const match = message.match(search.regexp);
```

Comment:
this should check all `searches` entries first, and use the nearest one (the smallest `message.indexOf(matched)` one),
otherwise URL before bug number will be overlooked.

Reply:
Good catch, thanks! Updated the code so it handles the closest match each time.